HybridCar.cpp

```diff
@@ -8,8 +8,13 @@ HybridCar::HybridCar(PetrolEngine* petrolEng, ElectricEngine* electricEng)
     std::cout << __FUNCTION__ << std::endl;
 }
 
-HybridCar::~HybridCar()       { std::cout << __FUNCTION__ << std::endl; }
+HybridCar::~HybridCar()
+{
+    delete ElectricCar::engine_;
+    delete PetrolCar::engine_;
```

Comment:
klase bazowe musza to posprzatac ;)

Reply:
Tak. Na zajęciach było, że destruktory klas bazowych będą wywołane. One sprzątają te dane i tutaj nic nie musimy robić :)

Reply:
poprawione ;)